components/brave_shields/browser/brave_shields_web_contents_observer.cc

```diff
@@ -105,6 +108,8 @@ std::map<BraveShieldsWebContentsObserver::RenderFrameIdKey, GURL>
     BraveShieldsWebContentsObserver::frame_key_to_tab_url_;
 std::map<int, GURL>
     BraveShieldsWebContentsObserver::frame_tree_node_id_to_tab_url_;
+const int kSiteSpecificScriptWorldID =
+  content::ISOLATED_WORLD_ID_CONTENT_END + 2;
```

Comment:
Google Translate uses END+1. It probably doesn't matter since we don't support Google Translate, but that's why I used +2 instead of +1. I couldn't find any other instances.

I'll move it to a header file.